### ISSUE TITLE
Demo Video

### ISSUE THREAD

**Issue by cmiya** [cwrc/HuViz#6]
PROBLEM:
- It is difficult (if not impossible) to learn how to navigate complex interfaces using only text-based instructions. Even screenshots have their limitations. A still image fails to capture the movement of the curser. Smaller steps – that seem obvious to the developer, but might not be intuitive for every user – can get cut, creating confusion. A demo video is the best way to capture the tool IN ACTION.
- The demo video also functions as promotional material for the tool itself. For this reason, it is important to keep it brief and lucid. Audio-visual quality matters. Still, it is possible to produce informative, high quality demo videos for relatively little effort.

SOLUTION:
- Create short, clear demo video (5 minutes max) geared to Novice Users (see Storify Example created by an undergraduate).

STEPS:
1) Decide on audience/goals (explain why people would want to use HuViz)
2) Prepare a script
3) Rehearse
4) Capture video from screen (using Quicktime or similar)
5) Record voiceover (best to do separately for better quality)
6) Edit together audio/video
7) Create titles
8) Export
9) Create a link to the video in the Documentation Tab, so that users could have the option to watch (or not watch)
10) Promote (post/re-post links on Twitter, CWRC, Orlando)

**Comment by antimony27** [comment]
@cmiya Are you working in CWRC now? We have a redirect for HuViz in CWRC, but I'm not clear if there is an info page yet. If so, we could start updating the tutorial, moving it over, etc.

**Comment by SusanBrown** [comment]
This is the stub I create a while ago. Chelsea and Mihaela were to meet about it but I don’t know if this happened. If you want to give me some additional content for it, I can upload it.

https://beta.cwrc.ca/huviz


On Jun 18, 2018, at 10:31 AM, antimony27 <notifications@github.com<mailto:notifications@github.com>> wrote:


@cmiya<https://github.com/cmiya> Are you working in CWRC now? We have a redirect for HuViz in CWRC, but I'm not clear if there is an info page yet. If so, we could start updating the tutorial, moving it over, etc.

**Comment by ilovan** [comment]
We did meet and, to my knowledge, Chelsea was working on it.

**Comment by cmiya** [comment]
Yep! @ilovan showed me how to edit the page. Here's the link(https://beta.cwrc.ca/huviz). I've created a list of content to add below. Is there anything I'm missing?

- A paragraph or two describing the tool
- List of publications/presentations *Do we have a list anywhere?
- Link to the GitHub page
- Credits?

**Comment by SusanBrown** [comment]
Agree to all above.

**Comment by SusanBrown** [comment]
No list of publications that I know of. Just trawled my CV quickly and plopped what I could find or know about into the page directly: https://beta.cwrc.ca/about/technical/huviz/publications

Needs details and consistent formatting. Anyone know of anything else? 

There was the INKE demo a couple of years back that you did, Chelsea. Anything like our poster that we want to put up online we can ingest into the CWRC publications collection in CWRC itself.
